app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyQuestionScreen.kt

```diff
@@ -17,22 +19,28 @@ import com.kks.nimblesurveyjetpackcompose.ui.theme.White50
 
 @Composable
 fun SurveyQuestionScreen(surveyQuestion: SurveyQuestion, pageNumber: Int, totalNumberOfPage: Int) {
-    Column(modifier = Modifier.fillMaxSize()) {
+    Column(modifier = Modifier
+        .fillMaxSize()
+        .padding(horizontal = 20.dp)) {
```

Comment:
Let's not forget about `cmd`+ `option` + `l` 😉 
```suggestion
    Column(
        modifier = Modifier
            .fillMaxSize()
            .padding(horizontal = 20.dp)
    ) {
```